src/EnergyPlus/WindowManager.cc

```diff
@@ -6097,7 +6097,7 @@ namespace WindowManager {
 			//  ( SurfaceWindow(SurfNum)%FrameEmis*(sigma*(SurfaceWindow(SurfNum)%FrameTempSurfIn+TKelvin)**4 - rmir) + &
 			//    hcin*(SurfaceWindow(SurfNum)%FrameTempSurfIn+TKelvin - tin) )
 
-			FrameHeatGain = SurfaceWindow( SurfNum ).FrameArea * ( 1.0 + SurfaceWindow( SurfNum ).ProjCorrFrIn ) * ( hcin * ( SurfaceWindow( SurfNum ).FrameTempSurfIn + TKelvin - tin ) );
+			FrameHeatGain = SurfaceWindow( SurfNum ).FrameArea * FrameCon * ( SurfaceWindow( SurfNum ).FrameTempSurfOut - SurfaceWindow( SurfNum ).FrameTempSurfIn );
```

Comment:
I think the FrameHeatGain should be left to match what is really coupled to the zone air.  If they don't agree then the zone gains don't match the sum of all the contributions.  You could revise the frame temp calculation if you want.  I would not expect the frame and divider to have the exact same IR exchange with glazing surface.

Maybe you could add a new output variable for whatever it is you want to find for the project.